help/unsubscribe-from-a-stream.md

```diff
@@ -4,19 +4,44 @@ You can always unsubscribe from any stream in Zulip.
 
 {start_tabs}
 
+{tab|desktop-web}
+
 {!stream-actions.md!}
 
 1. Click **Unsubscribe**.
 
 {end_tabs}
 
-## From the streams page (alternate method)
+## Alternate method to unsubscribe from a stream
+
+### Via the manage streams menu
 
 {start_tabs}
 
+{tab|desktop-web}
+
 {relative|stream|subscribed}
 
-1. Click on the green checkmark the left of the stream name.
+1. Click the green checkmark to the left of a stream to unsubscribe from it.
+
+{end_tabs}
+
+### Via the stream settings menu
+
+{start_tabs}
+
+{tab|desktop-web}
+
+{!stream-actions.md!}
+
+1. Click **Stream settings**.
+
+1. Click **Unsubscribe** near the top right corner of the menu.
```

Comment:
Again, I don't know that I would call this page a 'menu', so this goes for the headers and this instruction. Page? Panel? Overlay?

Reply:
How about "stream settings panel"?